avd_docs/aws/ec2/AVD-AWS-0164/docs.md

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Running `make docs` will generate the base documentation for the new check using the content from the Rule file.